packages/grafana-ui/src/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { FunctionComponent } from 'react';
+import { Range, createSliderWithTooltip } from 'rc-slider';
+import { css } from 'emotion';
+import { stylesFactory } from '../../themes';
+import { GrafanaTheme } from '@grafana/data';
+import { useTheme } from '../../themes/ThemeContext';
+
+export interface Props {
+  min: number;
+  max: number;
+  orientation: 'horizontal' | 'vertical';
```

Comment:
Can we make this optional and default to `horizontal`?

Reply:
Yes, great point. Updated!